source/includes/stackpath/_network_policy_rules.md

```diff
@@ -52,17 +52,22 @@ curl -X GET \
 
 Retrieve a list of all network policy rules in a given [environment](#administration-environments).
 
+
+Query Params | &nbsp;
+---- | -----------
+`workloadId`<br/>*string* | The workload ID to get the network policy rules for. It is optional.
+
 Attributes | &nbsp;
 ------- | -----------
-`id`<br/>*string* | The ID of the network policy rule, in the form `workloadId/hashCode`.
+`id`<br/>*string* | The ID of the network policy rule, in the form `networkProfileId/type/hashCode/occurrence`.
```

Comment:
Are we using `occurrence` as well, as part of network policy Id?

Reply:
yep.. if there is 2 records with the same hash. This avoid error in the client. That is part of my last commit on my PR